src/Mod/TechDraw/App/DrawProjectSplit.cpp

```diff
@@ -45,6 +44,7 @@
 #include <TopoDS.hxx>
 #include <TopoDS_Shape.hxx>
 #endif
+#include <BOPAlgo_Builder.hxx>
```

Comment:
This change should no longer be necessary, I've added that include to the PCH.

Reply:
Good. I hate adding changes to other people's code. I'll remove it.